src/OrchardCore/OrchardCore.Application.Cms.Core.Targets/OrchardCore.Application.Cms.Core.Targets.csproj

```diff
@@ -24,6 +24,7 @@
   -->
 
   <ItemGroup>
+    <ProjectReference Include="..\..\OrchardCore.Modules\OrchardCore.Metadata\OrchardCore.Metadata.csproj" />
```

Comment:
Move it in the section below, ordered alphabetically

Reply:
Please move the reference.